feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -130,6 +150,7 @@ fun Sessions(
 ) {
     val scheduleState = uiModel.state
     val isTimetable = uiModel.isTimetable
+    val timeLineState = rememberTimeLineState(uiModel.timeLine)
```

Comment:
I think this is the duplicate single source of truth. I thought we don't have to have the state in Compose. Is there any problem about this? 👀

Reply:
Fixed without using TimeLineState within Compose 🙇 

I was inexperienced about State of Compose.
So, I had put the TimeLine in the val property and was unable to update it, so I had to use State.
I was able to do it by using uiModel.timeLine directly.

* NG
```kotlin
@Composable
fun Sessions(
    uiModel: SessionsUiModel,
...
){
     val timeLine = uiModel.timeLine
     ...
    Timetable(
        timeLine = timeLine
    )
}
```

* OK
```kotlin
@Composable
fun Sessions(
    uiModel: SessionsUiModel,
...
){
     ...
    Timetable(
        timeLine = uiModel.timeLine
    )
}
```

Thanks for the advice!